### PR TITLE
Added Oreientation, Attribute related actions and specs

### PR DESCRIPTION
**Pull request by sravanmedarapu**
-  Updated NO_PROXY list and capabilities for the current session metadata

- Added attribute and Orientation related actions

- Added specs

### CODE REVIEW

File: lib/commands/general.js [hunk 1]
```diff
@@ -1,40 +1,17 @@
...
+  return await this.uiautomator2.jwproxy.command('/source', 'GET', {});
+};
 
 // Need to override this for correct unicode support
-commands.keys = async function (value) {
-  if (value instanceof Array) {
-    value = value.join("");
-  }
-  log.debug(`Setting text: '${value}'`);
-  value = encodeString(value, this.opts.unicodeKeyboard);
```
Comment (imurchie) [inline lib/commands/general.js]: Does this system not support Unicode? Or does it actually natively support it so we can do away with the ime stuff?

**Comment by imurchie** [comment]
Looks good.

File: lib/commands/general.js (hunk 1 above)
Comment (sravanmedarapu) [inline lib/commands/general.js]: Tried in multiple versions of devices and observed it has native support.

Comment (imurchie) [inline lib/commands/general.js]: That is awesome!

**Pull request merged**